values.yaml

```diff
@@ -343,6 +343,21 @@ server:
   # `server.ports.serflan.port` must be set to something other than 8301.
   exposeGossipAndRPCPorts: false
 
+  # [Enterprise Only] Configures the partitions and the port for each partition.
+  # Accepts an array of map with field name and port where where the name is the
```

Comment:
```suggestion
  # Accepts an array of map with field name and port where the name is the
```

Reply:
One other thought that crossed my mind, are there any specific caveats to the configurations below? Stuff like "ports must be different" or anything like that? Might be worth popping a comment in there with any?